include/nanobind/nb_eval.h

```diff
@@ -0,0 +1,69 @@
+/*
+    nanobind/nb_eval.h: Support for evaluating Python expressions and statements
+                        from strings and files
+
+    Adapted by Nico Schlömer from pybind11's eval.h with
+
+    Copyright (c) 2016 Klemens Morgenstern <klemens.morgenstern@ed-chemnitz.de> and
```

Comment:
At this point, the functionality is such a thin wrapper around stock Python functions that I don't think you need to include this extended copyright header. Maybe for the testcases part ("Testcases adapted from [..]"), which I now see you added already.